Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material

```diff
@@ -6,12 +6,11 @@
     "properties": {
         "baseColor": {
             "color": [
-                0.43414968252182007,
-                0.029556725174188614,
-                0.03955138474702835,
+                0.4005493223667145,
```

Comment:
I'm still not quite there yet.  Looking at https://www.babelcolor.com/index_htm_files/RGB%20Coordinates%20of%20the%20Macbeth%20ColorChecker.pdf if you convert the values to [0..1] range (divide by 255) the results are different.

Reply:
I plugged in the numbers for the 16 bit BabelColor average from https://www.babelcolor.com/index_htm_files/RGB%20Coordinates%20of%20the%20Macbeth%20ColorChecker.pdf using our own sRGB->Linear function, and the resulting values are:
0.43244872157
0.0297351510059
0.0399429307193
These should match perfectly with the tif.

Reply:
Plugged values- this is the result:
![Capture](https://user-images.githubusercontent.com/50616575/144898134-f431ba2b-0075-4492-b6f4-2062b1ccd73e.PNG)